### PR TITLE
added new 404 custom error page

### PR DESCRIPTION
**Pull request by NoLayerCode**
Hi,
I have added 404 custom error page in branch feature/404-page.

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link';
+import Footer from '@/components/footer';
+import Layout from '@/components/Layout';
+import styles from '@/components/Layout/Layout.module.scss';
+
+export default function FourOhFour() {
+    return <>
+        <Layout>
+            {/* <div className={styles.layout}> */}
```
Comment (whyDontI) [inline src/pages/404.js:9]: We don't commit commented code, can we remove this?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Good work @NoLayerCode 🙂,
Requested some changes.

**Comment by whyDontI** [comment]
Can you please fix the indentation with 2 spaces? 
Installing EsLint will also help

**Comment by NoLayerCode** [comment]
Hello @whyDontI ,
I have updated the changes as per the requirement please check.
And please do reply if any other changes are required

File: src/pages/404.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+import Layout from '@/components/Layout';
+import styles from '@/components/Layout/Layout.module.scss';
+
+export default function FourOhFour() {
+  return (
+    <>
+      <Layout>
+        <Link href="/">
+          {/* <a> */}
```
Comment (whyDontI) [inline src/pages/404.js:10]: Can we remove this commented code?

Comment (whyDontI) [reply src/pages/404.js:10]: Can we also have `cursor: pointer` while hovering on the text?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Great work @NoLayerCode 🥳
Requested some little changes.

File: src/pages/404.js (hunk 1 above)
Comment (harshith-venkatesh) [reply src/pages/404.js]: @NoLayerCode could you please work on this please

File: src/pages/404.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+import Layout from '@/components/Layout';
+import styles from '@/components/Layout/Layout.module.scss';
+
+export default function FourOhFour() {
```
Comment (sumitd94) [inline src/pages/404.js:5]: NIT: Can we please try renaming this to something like `PageNotFound`, this will sound better

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Requesting one small change :)

**Comment by NoLayerCode** [comment]
Hi,
I have updated the link cursor: pointer  issue and changed method name to PageNotFound

**Comment by NoLayerCode** [comment]
@whyDontI and @sumitd94 
If you have any other queries or requirements please do inform me

**Review by whyDontI — APPROVED** [review]
Nice work @NoLayerCode 🥳

**Pull request merged**